posts/scoping-plan-comments/emissions-table.js

```diff
@@ -0,0 +1,34 @@
+import { Table } from '@carbonplan/components'
+
+const EmissionsTable = ({ data }) => {
+  const titles = [
+    'Scenario',
+    'Net Zero date',
+    'CDR in 2045 (million tCO₂e)',
```

Comment:
It might be nice to render the units in `secondary` to make visually distinct from the `(selected)` text

Reply:
nice, will do!